src/api/ledger/parse/account-order.js

```diff
@@ -1,14 +1,27 @@
 'use strict';
-const parseOrderBase = require('./order-base');
+const utils = require('./utils');
+const flags = utils.core.Remote.flags.offer;
+const parseAmount = require('./amount');
 
 // rippled 'account_offers' returns a different format for orders than 'tx'
```

Comment:
Here are the two different values for indicating whether an offer is a sell, depending on whether it is for the transaction or for the ledger entry:

https://github.com/ripple/ripple-lib/blob/develop/src/core/transaction.js#L118
https://github.com/ripple/ripple-lib/blob/develop/src/core/remote.js#L220